### DISCUSSION TITLE
MS SQL SERVER not connecting to the budibase cloud.

### DISCUSSION THREAD

**Discussion by raj-potdar** [Budibase/budibase#7600] [Help]
I am trying to connect my MS SQL server in the budibase but there aren't allowing me to connect it to the budibase. I have whitelisted my IP on my server. Still I get an issue in connecting with the server. Are there any solutions for it? I am trying to connect it from two weeks and it's still giving me the same issue. There are no docs for how to connect the MS SQL to budibase.  I am using SQL SERVER 2019 
Please help.  @melohagan

**Comment by shogunpurple** [comment]
@raj-potdar please provide some more information:
- the error message you are seeing when you try to connect
- Any errors you see in the browser console when it occurs 
- where is your server hosted? Are the firewall rules definitely correct?
